### ISSUE TITLE
New version coming?

### ISSUE THREAD

**Issue by Tiberriver256** [hpsa/ADM-TFS-Extension#24]
@aslaakso - I see you are making changes here and in the MicroFocus repo. Does this mean a new version is going to get released soon? We're having some odd issues with the currently published version and a new release would be awesome!

**Comment by aslaakso** [comment]
Hi,

What kind of problems are you facing?

**Comment by Tiberriver256** [comment]
Just some odd results sometimes in the logs like this:

![image](https://user-images.githubusercontent.com/6989492/57939005-5e5c1a80-7897-11e9-9592-040d6b178a9f.png)

The entire task will show as succeeded though.

**Comment by aslaakso** [comment]
Try the updated version from here: https://marketplace.visualstudio.com/items?itemName=uftpublisher.application-UFT-TFS-Version2 .

If this does not fix your issue then contact our support and open a ticket.

Thanks.

**Comment by Tiberriver256** [comment]
Hmm... now I'm not getting any output in the VSTS logs. It just says `Started...` although I verified HpToolsLauncher.exe is running on the build server.

**Comment by aslaakso** [comment]
Did you update the HpToolsLauncher with the new one in the UFTWorking folder? Also PSModule.pdb and PSModule.dll.

Check the installation folder on github and there you will find the new content for the UFTWorking folder.

**Comment by Tiberriver256** [comment]
Yes, I tested with the latest UFTWorking folder. It seems to only be a problem with the new PSModule.dll if I use the new HpToolsLauncher.exe with the old bin folder it logs fine.

**Comment by aslaakso** [comment]
Did you uninstall completely the old extension before installing the new one or you installed  it over the old one?

**Comment by Tiberriver256** [comment]
I uninstalled it completely before installing the new one.
